frontend/src/fetchers/index.js

```diff
@@ -0,0 +1,24 @@
+export const get = async (url) => {
+  const res = await fetch(url, {
```

Comment:
This should be using axios for IE11, right?

Reply:
IE11 support for fetch (among other things) is handled by the [polyfill](https://github.com/HHS/Head-Start-TTADP/blob/ff221a79a71ad1f1a09f8bac1f145088db77d8e3/frontend/src/index.js#L1) (https://www.npmjs.com/package/react-app-polyfill)